src/test/scala/com/gd/twitteranalytics/TweetsDataLoaderTest.scala

```diff
@@ -0,0 +1,26 @@
+package com.gd.twitteranalytics
+
+import java.io.File
+import com.gd.twitteranalytics.util.AppConfigReader
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.scalatest.FreeSpec
+
+class TweetsDataLoaderTest extends FreeSpec with DataFrameSuiteBase {
+
+  "Methods tests" - {
+    "saveOutputToHdfs when presented with dataframe, savePath and partitionColumn" - {
+      "should create event_datetime folder structure in fileSystem at given path" in {
```

Comment:
no "event_date**time**" structure gets created.

Reply:
output format of files / content should be also validated, to ensure that the operation works as expected (durably persist the data and doesn't modify its content).
can be asserted in a different test case though.

Reply:
validating output format means that results should be able to get ridden by a parquet reader (spark.read.parquet...)

Reply:
Done.